submissions/lalalilulu/dom-api-menu/index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Меню</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body class="menu-task">
+<header class="header">
+    <div class="title-bar">
+        <h1 class="title">Уход за волосами</h1>
+    </div>
+</header>
+<div class="menu">
+    <aside class="menu-container">
+        <ul class="menu-list">
```

Comment:
It's not a blocker. You can't fix it. But next time you can make the whole your app data-driven. 
It means that no need to create html here, you can generate this all content from your `products ` list.